(fix) Design rejects - Part III

gabitoesmiapodo opened the pull request:
This PR is related to [#167347142](https://www.pivotaltracker.com/story/show/167347142)

Fixes or addresses the following issues from the **TO-DO** list:

- [x] Feed: Pending has it's own card design and modal design. (Orange) - need to wait for `#163177443` and  `#164289070` -> Modals will be addressed on https://www.pivotaltracker.com/story/show/167614001 / I'll check feed styling on another task.
- [x] Settings Menu: It's not the latest design. -> Will be worked on: #167646423
- [x] Fixed an issue where content would overflow from feed items on certain devices.
- [x] Feed: modal "from:" is too close to avatar image
- [x] claim screen doesn't fit iphone 5 -> **Here we could shrink some elements (ie: font-size, margins, etc.) but it would require altering what's in the design. Currently applied sizes are correct, so I don't think it would be wise for my to change all of that without asking you first if that's OK. Besides this particular issue, designs in general don't take into account the device's clock bar, browser top / bottom bar, etc., so some other section might not fit into an iPhone's 5 screen, even when at first glance the design's dimensions are OK.**
- [x] Send: "Enter the recipient name" should be in a brighter gray, please confirm all the colors. -> Changed the placeholder color.
- [x] Pop-up: The background should be brightened and blurred. -> Created an issue for this `#167653239`, not a trivial task.
- [x] Profile: The light blue icons are too small and not aligned. -> Fixed, and updated with new icons.
- [x] Profile: camera icon doesnt look centered -> Fixed, and updated with new icons.
- [x] Profile: There is still some issue with the profile pic, some sort of offset. -> There were some issues with the border and width, fixed them.

dminones commented on src/components/dashboard/FeedItems/FeedListItem.js:
```diff
@@ -45,6 +45,7 @@ const getStylesFromProps = ({ theme }) => ({
     elevation: 1,
     marginHorizontal: theme.sizes.default,
+    height: theme.feedItems.height,
     maxHeight: theme.feedItems.height,
```
If we are setting a fixed height. shouldn't we remove max and min height? 
Actually having a fixed height sounds like it should be avoidable, in the feed item. Perhaps that fixed height should be a consequence of inner sizes.

gabitoesmiapodo replied:
Yeah, that's there because the contents were overflowing the container on certain devices (and me trying to fix it).

I'm removing `minHeight` as it's redundant, but I think a fixed `height` is necessary / mandatory for this type of item, and I'm forcing a `maxHeight` equal to `height` to avoid content overflow.

It might have something to do with some `flexGrow: 1` or a similar property, but I don't want to delete that because I'm almost certain that other layout's behaviors depend on that. Code is a little touchy around there.

gabitoesmiapodo commented:
@fernandomg 

> I don't recall seeing this white margin/padding around the "Alpha tones are for..." poster, so I'm assuming this is a change introduced in this PR:

Fixed.

> I kinda see icons smaller than what they look in the wireframes. Also, it looks like the username has been changed in wireframes.

Yeah, I've noticed. I wasn't going to fix that as it wasn't reported on the TO-DO list, but it should be fixed now.

> In 'View Avatar' screen, buttons are unaligned, see wireframes. Also, found out that the title is different (it should now read: 'Your Profile'), and Profile reads 'Your Profile' also (this has nothing to do with the PR, but if you can change them to match the updated wireframes, that would be great). To make it to this page you have to tap on the avatar image in the profile page.

That section looks (almost) nothing like on the design, but I updated the icons and its positions anyway.
